doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -208,7 +208,7 @@ execute ::=
   'EXECUTE' name   ('(' (parameter_value) ( ',' parameter_value )* ')')?
 explain ::=
   'EXPLAIN'
-  ( ( 'RAW' | 'DECORRELATED' | 'OPTIMIZED' | 'PHYSICAL' ) 'PLAN' )?
+  ( ( ( 'RAW' | 'DECORRELATED' | 'OPTIMIZED' | 'PHYSICAL' ) 'PLAN' )? | 'TIMESTAMP' ? )
```

Comment:
I think we may have been leaving this out of the docs intentionally, @morsapaes do you know?

Reply:
There's a description of `explain timestamp` at the bottom of the `explain` page: https://materialize.com/docs/sql/explain/#timestamps

Reply:
Oh, should be fine then.